packages/@react-aria/table/src/useTableRow.ts

```diff
@@ -24,10 +24,12 @@ import {TableState} from '@react-stately/table';
 export function useTableRow<T>(props: GridRowProps<T>, state: TableState<T>, ref: RefObject<HTMLElement>): GridRowAria {
   let {node} = props;
   let {rowProps} = useGridRow<T, TableCollection<T>, TableState<T>>(props, state, ref);
+  // Not entirely sure why I needed this, but this fixed the tests... Weird cuz the aria-labelledby values didn't seem to be a problem in storybook
+  let rowLabel = useMemo(() => getRowLabelledBy(state, node.key), [state, node.key]);
```

Comment:
Still not sure why I had to do this :/

Reply:
were we relying on the now memo'ed selectionManager to get an extra render

Reply:
not sure, weird because the storybook stories didn't exhibit this issue